src/App.tsx

```diff
@@ -1,166 +1,15 @@
-import { Loader } from './components/Loader';
+import { Navbar } from './components/Navbar';
 
 import './App.scss';
+import { Outlet } from 'react-router-dom';
 
 export const App = () => (
   <div data-cy="app">
```

Comment:
You can additionaly add a html tag wrapper with bulma class 'has-navbar-fixed-top'. By doing this your application will look better as in a working version.
```suggestion
  <html className="has-navbar-fixed-top">
    <div data-cy="app">
```